iidm/iidm-api/src/main/java/com/powsybl/iidm/network/DanglingLine.java

```diff
@@ -92,6 +92,79 @@
      */
     DanglingLine setB(double b);
 
+    /**
+     * <p>Get the active power setpoint in MW.</p>
+     * <p>The active power setpoint follows a load sign convention.</p>
+     * <p>Depends on the working variant.</p>
+     * @return the reactive power setpoint
+     */
+    default double getActivePowerSetpoint() {
+        return Double.NaN;
+    }
+
+    /**
+     * <p>Set the active power setpoint in MW.</p>
+     * <p>Depends on the working variant.</p>
+     * @param activePowerSetpoint the active power setpoint
+     * @return this to allow method chaining
+     */
+    default DanglingLine setActivePowerSetpoint(double activePowerSetpoint) {
+        return this;
+    }
+
+    /**
+     * <p>Get the reactive power setpoint in MVAR.</p>
+     * <p>The reactive power setpoint follows a load sign convention.</p>
+     * <p>Depends on the working variant.</p>
+     * @return the reactive power setpoint
+     */
+    default double getReactivePowerSetpoint() {
+        return Double.NaN;
+    }
+
+    /**
+     * <p>Set the reactive power setpoint in MVAR.</p>
+     * <p>Depends on the working variant.</p>
+     * @param reactivePowerSetpoint the reactive power setpoint
+     * @return this to allow method chaining
+     */
+    default DanglingLine setReactivePowerSetpoint(double reactivePowerSetpoint) {
+        return this;
+    }
+
+    /**
+     * Get the voltage regulation status.
+     */
+    default boolean isVoltageRegulationOn() {
```

Comment:
Just a question, why do we use this and not something like `regulationType` that can be `OFF`, `VOLTAGE` and `REACTIVE`? Here when there is no voltage regulation, how can we know if there is an active or reactive regulation or not at all?

Reply:
@zamarrenolm What do you think?

Reply:
I think for dangling lines we should consider only voltage regulation, like in generators. In fact, we should duplicate the attributes from the IIDM generator into the dangling line, to highlight the fact that we are adding just an implicit/fictitious generator at the boundary end of the dangling line.